### DISCUSSION TITLE
What does the performance bar measure?

### DISCUSSION THREAD

**Discussion by enricojr01** [TechEmpower/FrameworkBenchmarks#7445] [Q&A]
Apologies in advance for the stupid sounding question, but what does the number on the performance bar on the rendered chart actually measure?

![image](https://user-images.githubusercontent.com/5615794/177084412-c70ab263-66e9-4d85-9ada-03df49038dda.png)

The value for `flask` is 9,868 but that's what? Requests per second? Or is it some synthetic score ala 3DMark? It's not exactly obvious , or at least I'm not seeing it.

**Comment by ishtms** [comment]
The numbers you're seeing is most probably requests per second. 

For the test you're viewing, Flask clocked at ~10k requests/sec.

**Comment by bhauer** [comment]
@ishtms is correct.

The text on a gold background above the chart describes what is being measured in the chart. For example, in the chart below, it's the best responses per second in the Single Query test type. The word "best" here means the best among multiple concurrency levels. You can see the results for each of the concurrency levels by switching to the "Data table" view (see the tabs above the gold bar).

![image](https://user-images.githubusercontent.com/777068/180582380-7446b38f-b3ba-45ca-b6ae-9a56cca24816.png)

Admittedly, the information density of the UI is fairly high and one might not even notice the text in the gold bar.  😄
